src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/probation/model/EnrichedProbationSearchResult.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.probation.model
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus
+import java.time.LocalDate
+
+data class EnrichedProbationSearchResult(
```

Comment:
sorry I meant under: `uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model`

Think it needs swagger annotations too?

Reply:
So this is now the model used for the transform to turn each of the entity search results into the model enriched search results. There is an existing model class in `uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model` called `ProbationSearchResult` which I have repurposed so it caters for a list of these model enriched search results with the two relevant counts. These have swagger annotation as well.